packages/api-reference/src/components/Section/SectionHeader.vue

```diff
@@ -12,17 +12,30 @@ withDefaults(
 </script>
 
 <template>
-  <LoadingSkeleton v-if="loading" />
-  <component
-    :is="`h${level}`"
-    v-else
-    class="section-header"
-    :class="{ tight }">
-    <slot />
-  </component>
+  <div class="section-header-wrapper">
+    <LoadingSkeleton v-if="loading" />
```

Comment:
To be honest I don't know if this skeleton actually works...

Reply:
I doubt it, too. Haven’t seen it in a while. 😅

Reply:
I just kept (moved) what we have nothing else, but we could ff and remove it if need

Reply:
Yeah sorry that wasn't an action item for you just me noticing it.